hooks/useMessages.tsx

```diff
@@ -0,0 +1,39 @@
+import { SUGGESTIONS } from "@/lib/consts";
+import trackNewMessage from "@/lib/stack/trackNewMessage";
+import { Message } from "ai";
+import { useState } from "react";
+import { Address } from "viem";
+import useUser from "./useUser";
+import { v4 as uuidV4 } from "uuid";
+
+const useMessages = () => {
```

Comment:
This hook seems to focus more on generating `suggestions` rather than `messages` right?
- actual: `useMessages`
- expected: `useSuggestions`